.github/workflows/release-and-publish.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: 1.18.2
+        go-version: 1.21
     # Check if the newest tag already exists
     - name: Check if tag exist
       uses: mukunku/tag-exists-action@5dfe2bf779fe5259360bb10b2041676713dcc8a3 # v1.1.0
```

Comment:
you need to bump the go mod. you don't do that anywhere

Reply:
also why not 22?

Reply:
the go.mod is already on 1.21

Reply:
where?

https://github.com/Azure/draft/blob/d64aa381436dccf0e9e468d33e32be57e59db76f/go.mod#L3

I see 1.18

Reply:
Tommy updated it in staging, we need to do the same for main next

Reply:
they've drifted since staging wasn't used for a while